.github/workflows/documentation.yml

```
@@ -24,11 +24,12 @@ jobs:
       - uses: julia-actions/cache@v2
       - uses: julia-actions/setup-julia@v2
         with:
-          version: '1.6'
+          version: '1'
```

```suggestion
          version: 'min'
          version: '1'
```
We want to test on the earliest supported version as well as on the latest release.